src/mainwindow.cpp

```diff
@@ -367,3 +373,36 @@ void MainWindow::aboutHotspot()
     dialog.adjustSize();
     dialog.exec();
 }
+
+void MainWindow::customContextMenu(const QPoint &point, QTreeView* view, int symbolRole)
+{
+    const auto index = view->indexAt(point);
+    if (!index.isValid()) {
```

Comment:
this is the correct way of doing it, the flamegraph should do the same (see comment above)